.prod/on_deploy.sh

```diff
@@ -2,3 +2,5 @@
 # Runs migration scripts on each deployment
 exec python /app/manage.py migrate --noinput
 
+# This can be removed once executed
+exec python /app/manage.py assign_ws_lease_stickers
```

Comment:
This won't ever be ran. The exec on line 3 will take over the shell, and the return value of that will be the whole return value of this script. If multiple commands need to be ran, then remove all execs.

Reply:
Good point! will fix it.